.buildkite/pipeline-raw.yml

```
@@ -154,6 +155,17 @@ steps:
      queue: "docker"
    only_changes: *native_packaging_changes_regexes
 
+ - label: test gen_systemd_service_file.py script
+   commands:
+   - eval "$SET_VERSION"
+   - nix run -f . pkgs.python3 -c ./gen_systemd_service_file.py tezos-node
```

That should work regardless nix version
```suggestion
   - nix-shell scripts/shell.nix --run "./gen_systemd_service_file.py tezos-node"
```